src/create-extension-pack.js

```diff
@@ -119,11 +134,27 @@ const repository = 'https://github.com/eclipse-theia/vscode-builtin-extensions';
             console.log('Adding: ' + extensionId);
             extensionsArr.push(extensionId);
         }
-        return Promise.resolve(extensionsArr);
+    }
+
+    async function resolveMsVscodeExtensions(extensionsArr) {
+        for (const msVscodeExtId of msVscodeBuiltins) {
```

Comment:
nitpick: we already know we are dealing with `msVscode` extensions in the method, we can therefore simplify it:

```suggestion
        for (const id of msVscodeBuiltins) {
```

or 

```suggestion
        for (const extensionId of msVscodeBuiltins) {
```

Reply:
Done, using `id`